### PR TITLE
Docs: cleanups & refactors & fixes

### PR DESCRIPTION
**Pull request by AlbertCarreras**
Docs: internal refactors

### CODE REVIEW

File: docs/pages/foundations/forms/available_components.js [hunk 1]
```diff
@@ -1,373 +1,191 @@
...
-          defaultCode={`
-          function CheckboxExample() {
-            const [checkedSell, setCheckedSell] = React.useState(false);
-            const [checkedLeads, setCheckedLeads] = React.useState(false);
-            const [checkedAudience, setCheckedAudience] = React.useState(false);
-            const [checkedBrand, setCheckedBrand] = React.useState(false);
-            const [checkedNotSure, setCheckedNotSure] = React.useState(false);
+        <MainSection.Subsection
+          title="CheckBox"
+          description={`SelCheckbox is used for multi-select.
```
Comment (dangerismycat) [inline docs/pages/foundations/forms/available_components.js:138]: typo: "Checkbox"

File: docs/pages/web/datapoint.js [hunk 2]
```diff
@@ -242,7 +242,6 @@ Be cautious of using Datapoint for large volumes of data. In cases where the con
         <MainSection.Subsection
           description={`The \`badge\` prop is available to add a [Badge](/web/badge) to the title. This feature is in beta, so please use cautiously.`}
```
Comment (dangerismycat) [inline docs/pages/web/datapoint.js:243]: `This feature is in beta, so please use cautiously.`

Should this sentence also be removed if we're removing the "Beta" badge?

**Review by dangerismycat — APPROVED** [review]

File: docs/pages/foundations/forms/available_components.js (hunk 1 above)
Comment (AlbertCarreras) [reply docs/pages/foundations/forms/available_components.js:138]: DONE

File: docs/pages/web/datapoint.js (hunk 2 above)
Comment (AlbertCarreras) [reply docs/pages/web/datapoint.js:243]: DONE

**Pull request merged**